openapi/data_repository_service.swagger.yaml

```diff
@@ -414,41 +386,32 @@ definitions:
       license:
         type: object
         description: License information for the exposed API
-  BundleObject:
+  ContentsObject:
```

Comment:
So, a `ContentsObject` is basically a 'reference' to a blob or bundle? Might be worth stating so in the description.